operator/Makefile

```diff
@@ -235,6 +235,12 @@ endef
 
 ##@ Development
 
+.PHONY: style
+style: generate manifests bundle ## Make sure generated files are up to date
+	echo 'Checking for diffs after making bundle, generate and manifests...'
+	echo 'If this fails, check if the invocation of the normalize-metadata.py script in Makefile'
+	echo 'needs to change due to formatting changes in the generated files.'
```

Comment:
```suggestion
	@echo 'Checking for diffs after making bundle, generate and manifests...'
	@echo 'If this fails, check if the invocation of the normalize-metadata.py script in Makefile'
	@echo 'needs to change due to formatting changes in the generated files.'
```

Reply:
Previously this message was only displayed if the `generate` and `manifests` targets caused no diffs *and* the `bundle` target caused some diffs.
Now it is now misleading, since it will steer people towards investigating `normalize-metadata.py` while the error will typically be more benign and caused by not committing changes produced by running `make generate manifests`.

Can you please split the `style` target to `style-manifests` and `style-bundle` with separate checks? It would also be great if the `If this fails, run `make -C operator manifests generate bundle` and commit the result.)` part of the message in the `.sh` file was moved to the target.

In my dream, these message would only be emitted only if the target is about to fail, rather than ever time, but I realize this might be asking for way too much. :-)